domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncService.cs

```diff
@@ -60,7 +56,12 @@ public class MeasurementsSyncService
 
             if (measurementsToPublish.Any())
             {
-                await IssueCertificate(measurementsToPublish, meteringPointType, gridArea, recipientId, technology, stoppingToken);
+                await IssueCertificates(measurementsToPublish,
```

Comment:
These could also just be syncInfo object :)

Reply:
done